Sources/MiniPlayer/AudioMiniPlayer.swift

```diff
@@ -180,6 +180,10 @@ private extension AudioMiniPlayer {
         artistLabel.text = metadata.artist
         if metadata.isAudioOnly {
             artworkImageView.image = metadata.artworkImage ?? UIImage(named: "no-artwork")
+            
+            if #available(iOS 11.0, *) {
+                artworkImageView.accessibilityIgnoresInvertColors = metadata.artworkImage != nil
```

Comment:
Shouldn't the `artworkImageView` always be set to true since in the case where `metadata.artworkImage` is nil, we are setting `artworkImageView.image` to a default artwork?

Reply:
I do not think so. The asset [no-artwork](https://github.com/videolan/vlc-ios/blob/d338742a878a6dda59e2fd7781097836802d955e/vlc-ios/Images.xcassets/MediaController/no-artwork.imageset/no-artwork%402x.png) is a plain image, inverting the colors should not be destructive on it.

Reply:
Indeed, I agree with you for the no-artwork!

Sadly I just thought of a special case. For the first version of the new UI we decided to use the AudioMiniPlayer as an VideoMiniPlayer as well! At least for now, this would means that we probably always need to set `accessibilityIgnoresInvertColors`.